contributing.md

```diff
@@ -56,6 +56,8 @@ cd lib/<package-name>
 npm run build:all
 // To run build only for a single package (not its dependencies)
 npm run build
+//If you run into an issue with shx, run the following
+npm install shx
```

Comment:
If there's issues related to dependencies not being installed let's make sure they are included in the dependency list rather than requiring one-off installs

Reply:
+1. Do we know why this is needed or rather when we run into issues without this install?

Reply:
I think maybe this should have been a global install? This is the first time Travis is using npm on his machine so it makes that it wouldn't be installed yet.

Reply:
Nevermind, shx didn't show up in my global install list. @trwalke what version of NodeJS and npm are you using?

Reply:
Should be installed as a devDependency in each lib. If it's not there and it's causing issues, please add it.

Reply:
@Robbie-Microsoft v18.18.2, the latest LTS version. Lets sync up on this so I can add it to the dependencies